src/libmongoc/doc/mongoc_database_create_collection.rst

```diff
@@ -25,7 +25,7 @@ Parameters
 Description
 -----------
 
-This function creates a :symbol:`mongoc_collection_t` from the given :symbol:`mongoc_database_t`.
+This function creates a :symbol:`mongoc_collection_t` from the given :symbol:`mongoc_database_t`. ``database`` must not be destroyed while the new :symbol:`mongoc_collection_t` is in use.
```

Comment:
Why do we need the `mongoc_collection_t` struct to depend on the `mongoc_database_t` struct? Here and in `src/libmongoc/doc/mongoc_database_get_collection.rst`

Reply:
It was so that collection could track changes in the parent db's timeout value to inherit them, but I've just removed that behavior so this dependency is gone, too.